tap_monday/streams/docs.py

```diff
@@ -11,4 +11,19 @@ class Docs(FullTableStream):
     replication_method = "FULL_TABLE"
     replication_keys = []
     data_key = "data.docs"
+    object_to_id = {"created_by": "creator"}
+    root_field = "docs(limit:{limit}, page:{page})"
+    page_size = 300
+    pagination_supported = True
+    extra_fields = {
+        "created_by": ["id"]
+        }
+    excluded_fields = ["creator_id"]
+
+    def update_data_payload(self, graphql_query: str = None, parent_obj: Dict = None, **kwargs) -> None:
+        """Update JSON body for GraphQL API. Injects query string if provided."""
+        page = kwargs.get("page", 1)
+        root_field = self.root_field.format(limit=self.page_size, page=page)
+        graphql_query = self.get_graphql_query(root_field)
```

Comment:
Why bracket has not been added here?

```suggestion
        graphql_query = self.get_graphql_query(root_field) + "}"
```